src/System.Management.Automation/engine/parser/TypeInferenceVisitor.cs

```diff
@@ -564,23 +564,23 @@ object ICustomAstVisitor.VisitHashtable(HashtableAst hashtableAst)
                             switch (expression)
                             {
                                 case ConstantExpressionAst constantExpression:
-                                {
-                                    value = constantExpression.Value;
-                                    break;
-                                }
+                                    {
+                                        value = constantExpression.Value;
+                                        break;
+                                    }
                                 default:
-                                {
-                                    typeName = InferTypes(kv.Item2).FirstOrDefault()?.Name;
-                                    if (typeName == null)
                                     {
```

Comment:
indentation here is also strange because the brace is indented, suggest removing brace

Reply:
Fixed.